server/util.go

```diff
@@ -333,3 +333,16 @@ func diffRegionKeyInfo(origin *RegionInfo, other *RegionInfo) string {
 
 	return strings.Join(ret, ",")
 }
+
+func parseTimestamp(data []byte) (time.Time, error) {
+	if data == nil {
+		return zeroTime, nil
+	}
```

Comment:
Unnecessary to check nil, _bytesToUint64_ takes good care of it.
BTW, prefer `if len(data) == 0` to `if data == nil` when check a slice.